pinot-core/src/main/java/org/apache/pinot/core/common/DataBlockCache.java

```diff
@@ -257,6 +313,17 @@ public int getNumDocs() {
     return intValues;
   }
 
+  /**
+   * Get the int[][] values for a single-valued column.
```

Comment:
Isn't this getting values for a multi-valued column? Same comment for Javadocs below.

Reply:
No: the JSON column is single value, but the JsonPath produces a multi-valued result. If it’s confusing the verbiage can be expanded upon.

Reply:
Just circling back here - this is ambiguous now the evaluator concept is more general than JSON, so I will remove the comment about the arity of the column.